acdc-ws/app/controllers/MetricController.scala

```diff
@@ -0,0 +1,23 @@
+package controllers
+
+import play.api.mvc._
+import services.Metric
+
+import javax.inject._
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class MetricController @Inject() (
```

Comment:
looks like this file need to be formatted too.

Reply:
I can do this.